.adr-dir

```diff
@@ -0,0 +1 @@
+doc/decisions
```

Comment:
I might prefer `doc/architecture/decisions` but not going to insist :)

Reply:
Maybe? I found it weird to make a directory that just contains another directory. (although doc does this, but at least that's top level).

It does seem more self-documenting though, maybe. @frankieroberto can make the tie-breaker.